### DISCUSSION TITLE
Fine-tuning dt by increasing or decreasing?

### DISCUSSION THREAD

**Discussion by peter-mitrano-bg** [NVlabs/curobo#279] [Research Q&A]
My understanding of the current method is to try decreasing the DT by some constant factor (0.85 by default?) and if that fails, keep increasing it until it succeeds, subject to a min dt of `interpolation_dt`. This does seem to reduce the DT in some cases, but it seems more intuitive to me to do the opposite -- reduce the DT until trajopt fails. Is there any mechanism in curobo for doing this? has anyone tried this and found downsides that I'm unaware of?

**Comment by balakumar-s** [comment]
You could change the for loop to reduce DT until trajopt fails. The caveat is that now your planning time will be much larger as in most common cases you will run more than 1 iteration of the for-loop.
